### PR TITLE
Add itoon channel

### PR DESCRIPTION
**Pull request by mrbooshehri**

### CODE REVIEW

**Review by fershady — COMMENTED** [review]
I have made a logo https://i.imgur.com/2HoOHwF.png

**Comment by MapGuy11** [comment]
@mrbooshehri ADD logo thanks then approve.

File: channels/ir.m3u [hunk 1]
```diff
@@ -152,6 +152,8 @@ http://51.210.199.57/hls/stream.m3u8
 http://159.69.58.154/irane_arya/irane_arya.m3u8
 #EXTINF:-1 tvg-id="ITN.ir" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Entertainment",ITN (720p) [Not 24/7]
 http://51.210.199.31/hls/stream.m3u8
+#EXTINF:-1 tvg-id="itoon" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Kids" (480p)
```
Comment (LaneSh4d0w) [inline channels/ir.m3u:155]:
```suggestion
#EXTINF:-1 tvg-id="itoon" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Kids" (480p), iToon
```

Missing part of identification.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
See attached commit suggestion.

File: channels/ir.m3u [hunk 2]
```diff
@@ -152,6 +152,8 @@ http://51.210.199.57/hls/stream.m3u8
 http://159.69.58.154/irane_arya/irane_arya.m3u8
 #EXTINF:-1 tvg-id="ITN.ir" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Entertainment",ITN (720p) [Not 24/7]
 http://51.210.199.31/hls/stream.m3u8
+#EXTINF:-1 tvg-id="itoon" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Kids", iToon (480p)
```
Comment (Dum4G) [inline channels/ir.m3u:155]:
```suggestion
#EXTINF:-1 tvg-id="iToon.ir" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Kids",iToon (480p)
```

**Review by Dum4G — DISMISSED** [review]

File: channels/ir.m3u [hunk 3]
```diff
@@ -152,6 +152,8 @@ http://51.210.199.57/hls/stream.m3u8
 http://159.69.58.154/irane_arya/irane_arya.m3u8
 #EXTINF:-1 tvg-id="ITN.ir" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Entertainment",ITN (720p) [Not 24/7]
 http://51.210.199.31/hls/stream.m3u8
+#EXTINF:-1 tvg-id="iToon.ir" tvg-country="IR" tvg-language="Persian" tvg-logo="" group-title="Kids",iToon (480p)
```
Comment (Dum4G) [inline channels/ir.m3u:155]:
```suggestion
#EXTINF:-1 tvg-id="iToon.ir" tvg-country="IR" tvg-language="Persian" tvg-logo="https://i.imgur.com/2HoOHwF.png" group-title="Kids",iToon (480p)
```

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**